constitution.md

```diff
@@ -162,15 +162,19 @@ the membership shall consider at a General Meeting whether that person's members
 
 ## 11 Functions of the Management Committee
 
-11.1 Except as otherwise provided by these rules and subject to resolutions of the members of the Society carried at any general meeting, the management committee:  
-* shall have the general control and management of the administration of the affairs and funds of the Society  
+11.1 Except as otherwise provided by these rules and subject to resolutions of the members of the Society carried at any general meeting, the management committee:
+* shall have the general control and management of the administration of the affairs and funds of the Society
 * shall have authority to interpret the meaning of these rules and any matter relating to the Society on which these rules are silent.
 
-11.2 The management committee may exercise all the power of the Society:  
+11.2 The management committee may exercise all the power of the Society:
 * to raise or secure the payment of money in such manner as the members of the Society may think fit and secure the payment or performance of any debt, liability or other engagement incurred or to be entered into by the Society in any way.
 
 11.3 The management committee will take full responsibility for all publications produced by the Society or by any of its members.
 
+11.4 At any event, function or gathering where UQCS is a financial contributor and pizza is offered, at least one (1) Hawaii\[a\]n pizza where:
+* a Hawaii\[a\]n pizza is a pizza originating in Canada, and is traditionally topped with pineapple, tomato sauce, cheese, and either ham or bacon [(source)](https://en.wikipedia.org/wiki/Hawaiian_pizza).
+* where procuring a Hawaii\[a\]n pizza from Canada is unfeasible, a substitute pizza originating from an establishment in Australia will suffice, provided that it meets the other requirements in 11.4.
```

Comment:
The phrase "a substitute pizza originating from an establishment in Australia will suffice, provided that it meets the other requirements in 11.4" is a bit concerning. There are no other requirements in 11.4, as the toppings listed are only "traditionally" supplied and the Canadian origin can be bypassed. Nothing is stopping UQCS executive from calling a meat-lovers pizza ("originating from an establishment in Australia") by the term "Hawaian".